### PR TITLE
OZ-17: Provide a lab order entry form for SENAITE

### PR DESCRIPTION
**Pull request by reagan-meant**

### CODE REVIEW

File: openmrs_config/ampathforms/blood.json [hunk 1]
```diff
@@ -0,0 +1,186 @@
+{
+    "name": "Blood Orders",
+    "description": "desssssss333",
```
Comment (Ruhanga) [inline openmrs_config/ampathforms/blood.json:3]: Do you intend to change this to something more descriptive?

File: openmrs_config/ampathforms/genitalswab.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
+{
+  "name": "Genitalswab Orders",
+  "description": "desssssss333",
```
Comment (Ruhanga) [inline openmrs_config/ampathforms/genitalswab.json:3]: Same here.

File: openmrs_config/ampathforms/skin.json [hunk 3]
```diff
@@ -0,0 +1,60 @@
+{
+  "name": "Skin Orders",
+  "description": "desssssss333",
```
Comment (Ruhanga) [inline openmrs_config/ampathforms/skin.json:3]: Same here.

File: openmrs_config/ampathforms/sputum.json [hunk 4]
```diff
@@ -0,0 +1,61 @@
+{
+  "name": "Sputum Orders",
+  "description": "desssssss333",
```
Comment (Ruhanga) [inline openmrs_config/ampathforms/sputum.json:3]: Same here.

File: openmrs_config/ampathforms/stool.json [hunk 5]
```diff
@@ -0,0 +1,68 @@
+{
+  "name": "Stool Orders",
+  "description": "desssssss333",
```
Comment (Ruhanga) [inline openmrs_config/ampathforms/stool.json:3]: Same here.

File: openmrs_config/ampathforms/urine.json [hunk 6]
```diff
@@ -0,0 +1,64 @@
+{
+  "name": "Urine Orders",
+  "description": "desssssss333",
```
Comment (Ruhanga) [inline openmrs_config/ampathforms/urine.json:3]: Same here.

**Review by Ruhanga — COMMENTED** [review]
Hi @reagan-meant, thanks for the PR. Except for a few comments I've added, when addressed would this PR be ready for merging in ?

**Review by mks-d — CHANGES_REQUESTED** [review]
@reagan-meant let's operate a separation of concerns as described here: https://issues.openmrs.org/browse/O3-858?focusedCommentId=283706&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-283706

**Pull request closed without merging**